docs/caching.md

```diff
@@ -158,10 +158,46 @@ In case you want to connect to a redis-cluster using IORedis's cluster functiona
                 host: 'localhost',
                 port: 7002,
             }
-        ],
+        ]
     }
 }
 ```
-Just specify all the nodes in the cluster inside an array with their hosts  and ports.
+
+or
+
+```typescript
+{
+    type: "mysql",
+    host: "localhost",
+    username: "test",
+    cache: {
+        type: "ioredis/cluster",
+        options: {
+            startupNodes: [
+                {
+                    host: 'localhost',
+                    port: 7000,
+                },
+                {
+                    host: 'localhost',
+                    port: 7001,
+                },
+                {
+                    host: 'localhost',
+                    port: 7002,
+                }
+            ],
+            options: {
+                scaleReads: 'all',
+                clusterRetryStrategy: function (times) { return null },
+                redisOptions: {
+                    maxRetriesPerRequest: 1
+                }
+            }
+        }
+    }
+}
+```
+Just specify all the nodes in the cluster inside an array with their hosts and ports.
```

Comment:
This sentence after object config is really confusing. Also object based configuration should be first (as it provide more options) and array based config should be only as simple note with description.